src/column.rs

```diff
@@ -33,21 +34,48 @@ impl Display for FunctionExpression {
             FunctionExpression::GroupConcat(ref col, ref s) => {
                 write!(f, "group_concat({}, {})", col, s)
             }
+            FunctionExpression::Generic(ref name, ref args) => write!(f, "{}({})", name, args),
         }
     }
 }
 
 #[derive(Clone, Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
-pub enum FunctionArguments {
+pub struct FunctionArguments {
+    pub arguments: Vec<FunctionArgument>,
+}
+
+impl Display for FunctionArguments {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let mut first = true;
+        for argument in self.arguments.iter() {
```

Comment:
You can use https://doc.rust-lang.org/stable/std/primitive.slice.html#method.join here to simplify this code

Reply:
Done